site/_resources/kubernetes.md

```diff
@@ -11,13 +11,14 @@ The `client-go` package includes a [compatibility matrix][2] as to what Kubernet
 
 ## Supported Kubernetes versions
 
-| Kubernetes version | Contour v1.0.0 | Contour v1.0.1 | Contour v1.1.0 |
-| ------------ | :-----------: | :-----------: | :----------: | 
-| 1.15.x | Supported | Supported | Supported | 
-| 1.16.x | Supported | Supported | Supported |
-| 1.17.x | Supported | Supported | Supported | 
-| 1.18.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Not Supported <sup>1</sup> |
+| Kubernetes version | Contour v1.0.0 | Contour v1.0.1 | Contour v1.1.0 | Contour v1.2.0 |
+| ------------ | :-----------: | :-----------: | :----------: | :--------: |
+| 1.15.x | Supported | Supported | Supported | Supported |
+| 1.16.x | Supported | Supported | Supported | Supported |
+| 1.17.x | Supported | Supported | Supported | Supported |
+| 1.18.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Not Supported <sup>1</sup> | Not Supported <sup>1</sup> |
 
+<br>
```

Comment:
please delete this, mixing html and markdown makes Jekyll sad.

Reply:
It works in Netlify just fine: https://deploy-preview-2263--projectcontour.netlify.com/resources/kubernetes/

Where does the sadness come from?

Reply:
See #2234 #2252

Reply:
Also `<br>` adds a line break inside html, did you mean `<hr>`?

Reply:
I just wanted to add some space after the table. I'll delete the `<br>` but it looks fine in Netlify and my local build.